.github/workflows/composites/build-integration-tests-project/action.yaml

```diff
@@ -13,5 +13,5 @@ runs:
         
         cd spring-cloud-kubernetes-integration-tests
         # build the images, but dont run the tests
-        .././mvnw -T 1C clean install -DskipTests
+        .././mvnw -T 1C clean install -nsu -DskipTests
```

Comment:
Are you saying its pulling down a snapshot version from the wrong branch?

Reply:
Well then there is something else that is wrong then because it at the very least it should be pulling the version for the branch being built not a different version.

Reply:
So it wasn't that we were pulling the wrong version of the dependencies, its that the cached version was using what was coming from spring snapshots and not what was built by the build?